src/NewBookForm/NewBookForm.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+
+export const NewBookForm = ({ setDisplay }) => {
+    const [title, setTitle] = useState('');
+    const [pages, setPages] = useState(0);
+    return(
+        <form>
+          <label>
+          What's the title of your book?
+            <input
+              aria-label="title input"
+              className="title-input"
+              placeholder="tell me your book title"
+              value={title}
+              onChange={event => setTitle(event.target.value)}>
+            </input>
+          </label>
+          <label>
+          How many pages does your book have?
+            <input
+              aria-label="pages input"
+              className="pages-input"
+              placeholder="tell me how many pages there are"
+              value={pages}
+              onChange={event => setPages(event.target.value)}>
+            </input>
```

Comment:
can return this JSX in a `.map` to DRY it up (especially when we have more inputs)